templates/object-cache.php

```
@@ -327,65 +327,136 @@ private function connect_redis() {
-						// Grab a random replica.
-						$replica = $config['replicas'][ array_rand( $config['replicas'] ) ];
-						$r_host  = isset( $replica['host'] ) ? $replica['host'] : '127.0.0.1';
-						$r_port  = isset( $replica['port'] ) ? (int) $replica['port'] : 6379;
+					foreach ( $nodes as $node ) {
+						list( $s_host, $s_port ) = array_pad( explode( ':', $node ), 2, 26379 );
 						try {
-							$tmp_replica = new \Redis();
-							if ( $tmp_replica->connect( $r_host, $r_port, 1.0 ) ) {
-								if ( ! empty( $replica['password'] ) ) {
-									$tmp_replica->auth( $replica['password'] );
+							$sentinel = new \RedisSentinel(
```

PHPStan (2.1.46)

[error] Class RedisSentinel constructor invoked with 1 parameter, 2-6 required.


(arguments.count)